models/ldndc/DESCRIPTION

```diff
@@ -7,14 +7,15 @@ Authors@R: c(person("Henri", "Kajasilta", role = c("aut", "cre"),
                     email = "henri.kajasilta@fmi.fi"))
 Description: This module provides functions to link the (LDNDC) to PEcAn.
 Imports:
-    PEcAn.logger,
-    PEcAn.utils (>= 1.4.8),
+    data.table,
```

Comment:
I realize that here you're just sorting existing imports, but while I'm looking: 
Am I correct that PEcAn.LDNDC uses `data.table` only for `fwrite`? How difficult would it be to switch that to a base or `readr` approach and remove the import of data.table?

Reply:
Yes, `data.table` is only used for `fwrite`. I think `readr` will work fine ( `readr::write_delim` ) and we can remove the import!